third_party/terraform/tests/resource_bigtable_app_profile_test.go

```diff
@@ -8,6 +8,8 @@ import (
 )
 
 func TestAccBigtableAppProfile_update(t *testing.T) {
+	// bigtable does not use the shared HTTP client
```

Comment:
do you also need a skip on the app profile generated tests?
(also the comment is _mostly_ true, but this resource and the IAM ones I think do use the shared HTTP client, it's just that their tests also create instances, which don't)

Reply:
Yeah, app profile tests set up an instance. I'll change the note to reference the bigtable instance only